frontend/__tests__/containers/ConfigStep/Board.test.tsx

```diff
@@ -8,36 +8,51 @@ import {
   RESET,
   VERIFIED,
   VERIFY,
-  VERIFY_ERROR_MESSAGE,
-  VERIFY_FAILED,
 } from '../../fixtures';
-import { fireEvent, render, screen, waitFor, within } from '@testing-library/react';
+import {
+  fireEvent,
+  getByLabelText,
+  render,
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+  within,
+} from '@testing-library/react';
 import { Board } from '@src/containers/ConfigStep/Board';
 import { setupStore } from '../../utils/setupStoreUtil';
+import userEvent from '@testing-library/user-event';
 import { Provider } from 'react-redux';
 import { setupServer } from 'msw/node';
 import { HttpStatusCode } from 'axios';
 import { rest } from 'msw';
-import React from 'react';
-
-export const fillBoardFieldsInformation = () => {
-  const fields = ['Board Id', 'Email', 'Project Key', 'Site', 'Token'];
-  const mockInfo = ['2', 'mockEmail@qq.com', 'mockKey', '1', 'mockToken'];
-  const fieldInputs = fields.map((label) => screen.getByTestId(label).querySelector('input') as HTMLInputElement);
-  fieldInputs.map((input, index) => {
-    fireEvent.change(input, { target: { value: mockInfo[index] } });
-  });
-  fieldInputs.map((input, index) => {
-    expect(input.value).toEqual(mockInfo[index]);
-  });
+
+export const fillBoardFieldsInformation = async () => {
+  await userEvent.type(screen.getByLabelText(/board id/i), '1');
+  await userEvent.type(screen.getByLabelText(/email/i), 'fake@qq.com');
+  await userEvent.type(screen.getByLabelText(/site/i), 'fake');
+  await userEvent.type(screen.getByLabelText(/token/i), 'fake-token');
```

Comment:
define a FAKE_TOKEN in the fixture, and refactoe existing code thx

Reply:
fixed